# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBuiltins.h
# Commit: 349728105c40
# Full Hash: 349728105c40993896170f4c2f43e152a3466706
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-01-08 09:40:24
# Description:
#   Bug 1748602 - Backout bug 1642412. r=lth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D135346
# ==============================================================================

diff -r 32873022d2ed -r 349728105c40 js/src/wasm/WasmBuiltins.h
--- a/js/src/wasm/WasmBuiltins.h	Fri Jan 07 20:35:15 2022 +0000
+++ b/js/src/wasm/WasmBuiltins.h	Fri Jan 07 20:43:16 2022 +0000
@@ -111,10 +111,11 @@
   TableCopy,
   ElemDrop,
   TableFill,
-  TableGetFunc,
+  TableGet,
   TableGrow,
   TableInit,
-  TableSetFunc,
+  TableSet,
+  TableSize,
   RefFunc,
   RefTest,
   RttSub,
@@ -236,10 +237,11 @@
 extern const SymbolicAddressSignature SASigTableCopy;
 extern const SymbolicAddressSignature SASigElemDrop;
 extern const SymbolicAddressSignature SASigTableFill;
-extern const SymbolicAddressSignature SASigTableGetFunc;
+extern const SymbolicAddressSignature SASigTableGet;
 extern const SymbolicAddressSignature SASigTableGrow;
 extern const SymbolicAddressSignature SASigTableInit;
-extern const SymbolicAddressSignature SASigTableSetFunc;
+extern const SymbolicAddressSignature SASigTableSet;
+extern const SymbolicAddressSignature SASigTableSize;
 extern const SymbolicAddressSignature SASigRefFunc;
 extern const SymbolicAddressSignature SASigPreBarrierFiltering;
 extern const SymbolicAddressSignature SASigPostBarrier;